packages/core/src/state-management/__tests__/cache-eviction.test.ts

```diff
@@ -191,10 +175,10 @@ describe('evicted then subscribed', () => {
   });
 
   test('pinned', async () => {
-    const stream1 = await TileDocument.create(ceramic, null, metadata, { syncTimeoutSeconds: 0 });
+    const stream1 = await TileDocument.create(ceramic, { foo: Math.random().toString() }, null, { syncTimeoutSeconds: 0 });
```

Comment:
nitpick: you don't really need the Math.random since you're not making a deterministic document anymore, the document will automatically have a random number added to the metadata to make it unique

Reply:
Yes, makes sense. Fixing that.

Reply:
Changing it to `await TileDocument.create(ceramic, { foo: 'bar' })` since `syncTimeoutSeconds` gets defaulted to 0 when `deterministic` is absent (i.e. no `metadata` is passed) or is `false`.

Reply:
I guess that applies elsewhere in this file as well lol, but I'll come around to those tests some other time :)